luigi_pipeline/lib/hail_vep_runners.py

```diff
@@ -115,8 +115,8 @@ class HailVEPDummyRunner(HailVEPRunnerBase):
                                                              'hgvsp': 'ENSP00000317992.6:p.Ser556Leu',
                                                              'impact': 'MODERATE',
                                                              'intron': hl.null('str'),
-                                                             'lof': hl.null('str'),
-                                                             'lof_filter': hl.null('str'),
+                                                             'lof': 'LC',
+                                                             'lof_filter': 'END_TRUNC,INCOMPLETE_CDS',
```

Comment:
hm... actually, I'm not 100% sure this is the right format, gonna go double check.

Reply:
Ok, I went into `gnomad_v3` and poked around, and I'm fairly confident that this format is correct.